src/Microsoft.Framework.PackageManager/Restore/NuGet/HttpSource.cs

```diff
@@ -131,13 +139,20 @@ internal class HttpSource
 
                 using (var response = await _client.SendAsync(request, HttpCompletionOption.ResponseHeadersRead))
                 {
+                    if (!throwNotFound && response.StatusCode == HttpStatusCode.NotFound)
```

Comment:
Think about this scenario:
You have two feeds in NuGet.config: FeedA and FeedB
100 packages your app needs are only available on FeedA and 1 package your app needs is only available on FeedB.
Do we want to fail and retry the 100 packages on FeedB even they are not there?